pytorch/training/docker/1.12/py3/cu113/Dockerfile.gpu

```diff
@@ -62,7 +62,7 @@ ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
 ENV CUDNN_VERSION=8.2.0.53
 ENV NCCL_VERSION=2.10.3
 ENV HOROVOD_VERSION=0.24.3
-ENV EFA_VERSION=1.15.1
+ENV EFA_VERSION=1.18.0
```

Comment:
1.19.0 is available now

Reply:
Do we want to double-click on EFA version with the platform team? They said that AL2 migration would use 1.18.0

Reply:
yes, please double check the version is consistent